dep_updates.md

```diff
@@ -44,12 +46,8 @@ You will need to sign in to DockerHub and you may also need to make DockerHub do
 - https://hub.docker.com/repository/registry-1.docker.io/ld4p/sinopia_exporter/tags?page=1&ordering=last_updated
 - https://hub.docker.com/repository/registry-1.docker.io/ld4p/sinopia_indexing_pipeline/tags?page=1&ordering=last_updated
 
-### Create a terraform-aws PR for Ops
+### Create a terraform-aws PR
```

Comment:
I will not be deploying my own terraform-aws PR.   I touch terraform maybe every 8 months in a cursory way, and I log into my aws console even less.

Reply:
Two of the projects that we support use AWS and terraform. Since we are all first responders, every team member should be able to log into AWS and deploy via terraform.

Is this a huge pain? Yes. I regularly have to look up the role names for AWS and troubleshoot my terraform install. But those are the tools that we have today. And having to do this when deploying dependency updates forces us to keep our environments current.

Reply:
I've been thinking that "ask ops to deploy to all environments" is sufficient but maybe that is flawed. Good to discuss further.

Reply:
personally, i've tended to use FR deployments as a chance to make sure my terraform setup was still working.  if i'm otherwise swamped and something isn't working with it, i will tend to just ask ops to deploy it all.  but if possible, i try to do the `terraform plan` and `terraform apply` for stage.

mostly because it seems like the expectation is that we will have to do occasional terraform and AWS console stuff for the foreseeable future?

(but it has been many many months since i've logged into my AWS console account 🙂)